alerts/kube_apiserver.libsonnet

```diff
@@ -103,6 +105,20 @@ local utils = import 'utils.libsonnet';
             componentName:: 'KubeAPI',
             selector:: $._config.kubeApiserverSelector,
           },
+          {
+            alert: 'KubeAPITerminatedRequests',
+            expr: |||
+              sum(rate(apiserver_request_terminations_total{%(kubeApiserverRequestTerminationsSelector)s}[10m]))  / (  sum(rate(apiserver_request_total{%(kubeApiserverRequestTerminationsSelector)s}[10m])) + on(apiserver,resource,verbs,service,namespace) sum(rate(apiserver_request_terminations_total{%(kubeApiserverRequestTerminationsSelector)s}[10m]))  ) > %(kubeAPIRequestTerminationsPercent)s
```

Comment:
I don't believe the `on(...)` is necessary here, is it?

Reply:
Yes, you are right! The `on(...)` clause isn't needed.